src/snark/libsnark/algebra/fields/bigint.tcc

```diff
@@ -124,7 +124,7 @@ size_t bigint<n>::num_bits() const
         }
         else
         {
```

Comment:
Add ``static_assert(GMP_NUMB_MAX <= ULLONG_MAX, "coercing limb to unsigned long long might truncate");``